packages/vscode-extension/src/dependency/DependencyManager.ts

```diff
@@ -167,10 +167,14 @@ export class DependencyManager implements Disposable, DependencyManagerInterface
     try {
       const env = getLaunchConfiguration().env;
       const shouldUseBundle = await this.shouldUseBundleCommand();
-      const process = command(shouldUseBundle ? "bundle exec pod install" : "pod install", {
-        cwd: iosDirPath,
-        env: { ...env, LANG: "en_US.UTF-8" },
-      });
+      const process = command(
+        shouldUseBundle ? "bundle install && bundle exec pod install" : "pod install",
+        {
+          shell: shouldUseBundle, // when using bundle, we need shell to run multiple commands
+          cwd: iosDirPath,
+          env: { ...env, LANG: "en_US.UTF-8" },
```

Comment:
Would be nice to add LANG automatically.

Reply:
I don't think we actually want to add it in all scenarios. I think we analyze the printed output here and so we add it such that we don't get the error messages translated.